### PR TITLE
changes in multiselect

### PR DESCRIPTION
**Pull request by Indra-1996**

### CODE REVIEW

File: src/components/Select/index.js [hunk 1]
```diff
@@ -32,32 +34,62 @@ export const Select = React.forwardRef(function Select(
...
+          className={['contacto-multi-select']}
+          showArrow={false}
+          ref={ref}
+          disabled={readOnly || disabled}
+          listHeight={listHeight || 220}
+          placeholder={placeholder}
+          dropdownClassName={['sg contacto-select-listbox', dropdownClassName].join(' ')}
+          mode={mode}
+          tagRender={
+            mode === 'multiple'
```
Comment (samuellawerentz) [inline src/components/Select/index.js:75]: I guess this condition is unnecessary as we already have a parent condition that checks for the mode.

File: src/components/Select/select.scss [hunk 2]
```diff
@@ -91,6 +91,78 @@
...
+        box-shadow: 0px 0px 8px rgba(106, 110, 128, 0.15);
+        border: 1px solid var(--border-color-1);
+        height: var(--height);
+        width: 100%;
+
+        &--no-shadow {
+            box-shadow: none;
+        }
+
+        .close-tag-icon {
```
Comment (samuellawerentz) [inline src/components/Select/select.scss:105]: Is this style is related to Tag component? If yes, can we please check if we can move it to Tag.scss file?

**Review by samuellawerentz — COMMENTED** [review]
Great Work!

File: src/components/Select/index.js (hunk 1 above)
Comment (Indra-1996) [reply src/components/Select/index.js:75]: Yes, I will remove it

**Review by samuellawerentz — APPROVED** [review]
